### PR TITLE
Monitoring by the Independent Stack

### PR DESCRIPTION
**Pull request by bastjan**
## Checklist

- [x] PR contains a single logical change (to build a better changelog).
- [x] Update the documentation.
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.

### CODE REVIEW

File: class/defaults.yml [hunk 1]
```diff
@@ -258,3 +258,7 @@ parameters:
                 # How many nodes need to be unused.
                 # There should be some overcapacity to account for failing nodes and future growth.
                 reserved: 4
+
+    independent_monitoring:
```
Comment (bastjan) [inline class/defaults.yml:262]: `syn_monitoring`?

File: tests/golden/release-4.10/openshift4-monitoring/openshift4-monitoring/prometheus_rules.yaml [hunk 2]
```diff
@@ -2284,11 +2284,10 @@ spec:
               }}.'
             summary: etcd cluster has high number of failed grpc requests.
             syn_component: openshift4-monitoring
-          expr: "100 * sum(rate(grpc_server_handled_total{job=\"etcd\", grpc_code=~\"\
```
Comment (bastjan) [inline tests/golden/release-4.10/openshift4-monitoring/openshift4-monitoring/prometheus_rules.yaml:2287]: Upstream manifest changes

File: component/independent-monitoring.libsonnet [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+local promInstance =
+  if params.independent_monitoring.instance != null then
+    params.independent_monitoring.instance
+  else
+    inv.parameters.prometheus.defaultInstance;
+
+local endpointConfiguration = function(serverName) {
+  bearerTokenSecret: {
+    key: '',
+  },
```
Comment (simu) [inline component/independent-monitoring.libsonnet:21]: This is not necessary.
```suggestion
```

File: class/defaults.yml (hunk 1 above)
Comment (simu) [reply class/defaults.yml:262]: I'd probably use `syn_monitoring` here

**Review by simu — APPROVED** [review]
LGTM overall

**Pull request merged**